databricks/koalas/tests/test_series.py

```diff
@@ -477,3 +477,29 @@ def test_cumsum(self):
     def test_median(self):
         with self.assertRaisesRegex(ValueError, "accuracy must be an integer; however"):
             koalas.Series([24., 21., 25., 33., 26.]).median(accuracy="a")
+
+    def test_rank(self):
+        pser = pd.Series([1, np.nan, 2, 3, 1], name='x')
+        kser = koalas.from_pandas(pser)
+        self.assert_eq(repr(pser.rank(na_option='keep')),
```

Comment:
Let's use `assertEqual` instead of `assert_eq`. `assert_eq` compares pandas dataframes too but we only need to compare the strings here.